.travis.yml

```diff
@@ -10,6 +10,12 @@ os:
 cache:
   directories:
     - $HOME/.npm
+    - $HOME/.deno
```

Comment:
Feel free to remove this file completely, I tried migrating to travis dot com but looks like they no longer give easy free CI for OSS, GitHub Actions is the way to go, thanks!
Also would be great to update links/badges in readme.